internal/codegen/stencil_test.go

```diff
@@ -121,3 +122,34 @@ func ExampleStencil_PostRun() {
 	// Output:
 	// hello
 }
+
+func TestStencilPostRunError(t *testing.T) {
+	const (
+		name    = "TestStencilPostRunError"
+		command = "invalidPostRunCommand"
+	)
+
+	var (
+		ctx              = context.Background()
+		logger           = logrus.New()
+		manifestContents = fmt.Sprintf("name: %s\npostRunCommand:\n- command: %s\n", name, command)
+		errMsg           = fmt.Sprintf("failed to run post run command for module %s and command %s", name, command)
+	)
+
+	fs := createFakeModuleFSWithManifest(t, manifestContents)
+	st := NewStencil(&configuration.ServiceManifest{Name: name},
+		[]*modules.Module{modules.NewWithFS(ctx, name, fs)}, logger)
+
+	assert.ErrorContains(t, st.PostRun(ctx, logger), errMsg)
+}
+
+func createFakeModuleFSWithManifest(t *testing.T, manifestContents string) billy.Filesystem {
```

Comment:
suggestion: please add a doc comment for this function and also use it in the other two tests that write manifest files.